accounting/accounting_cli.py

```diff
@@ -47,22 +47,25 @@ def main():
         "--username", help="username", metavar="USERNAME",
     )
     subparser_add_user.add_argument(
-        "--admin-level", help="admin level", metavar="ADMIN_LEVEL",
+        "--admin-level", help="admin level", default=1, metavar="ADMIN_LEVEL",
```

Comment:
Does level 1 mean the adding user is a regular user? If that's the case, this is okay.If 1 means admin, perhaps we should change this value to be a normal user.

Reply:
Good question. Yes, level 1 is a regular user. The higher the integer, the higher the privileges (which is behavior I think I just emulated from Slurm).